superset/utils/logging_configurator.py

```diff
@@ -41,19 +41,11 @@ def configure_logging(
         if app_config["SILENCE_FAB"]:
             logging.getLogger("flask_appbuilder").setLevel(logging.ERROR)
 
-        # configure superset app logger
-        superset_logger = logging.getLogger("superset")
```

Comment:
I have to admit I don't like this section that ties a bunch of behaviors to "debug_mode" potentially overriding `LOG_LEVEL`. It's like some configuration flags influence one another in a way that admins can't control. 

I feel this needs more thinking/refactoring. 

One option would be to move the DEBUG/LOG_LEVEL entanglement into `superset/config.py` so people can see that day `LOG_LEVEL = logging.DEBUG if DEBUG else logging.INFO` and choose to override it in their `superset_config.py`.

Reply:
Yes, I think that's a very good idea. It is technically a breaking change, but only for debug-level environments so should be acceptable.

Reply:
(I can move this addition to a separate PR if it requires further discussion.)